src/main/webapp/static/scripts/find-mentor.js

```diff
@@ -88,7 +88,16 @@ for (const buttonName in buttonDict) {
       if (mentorCardContainer.childElementCount < MIN_MENTOR_THRESHOLD) {
         refillMentors();
       }
+
       getActiveMentorCard();
     }
   });
 }
+
+const updateButtonContainer = () => {
```

Comment:
maybe call this method every time a rating is sent. that way, once they run out, they get the same update. also maybe toggle visibility of this div and reveal a second div witht he warning. that way we can bring the buttons back if something changes

Reply:
This function is called onload so the buttons should come back when stuff changes and mentors appear. Calling this method every time a rating is sent doesn't work if the user refreshes the page after seeing the message (the button reappears, which is not intended).